runtime/compiler/runtime/SignalHandler.c

```diff
@@ -337,86 +337,88 @@ UDATA jitX86Handler(J9VMThread* vmThread, U_32 sigType, void* sigInfo)
 		} else {
 			void * stackMap;
 			UDATA registerMap;
-
-			switch (sigType) {
-			case J9PORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO:
-			case J9PORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO:
-				if (0xF7 == *eip ) {
-					/* find the divisor */
-					UDATA div = jitX86decodeIdivInstruction(PORTLIB, sigInfo, eip);
-					if (div != 0 ) {
-						*eipPtr += jitX86decodeInstruction(eip);
-						*eaxPtr = 0x80000000;
-						*edxPtr = 0;
-						return J9PORT_SIG_EXCEPTION_CONTINUE_EXECUTION;
+			BOOLEAN isCFGEnabled = (J9_ARE_ANY_BITS_SET(vmThread->javaVM->sigFlags, J9_SIG_WINDOWS_MITIGATION_POLICY_CFG_ENABLED));
+
+			if(!isCFGEnabled) {
```

Comment:
Please remove the outermost parentheses on line 340 and add a space before `(` on line 342.
Same comments for lines 1782-1784.

Reply:
Addressed in [f487b7d](https://github.com/eclipse-openj9/openj9/commit/f487b7debac3f68af165f4b3c951f67811140d65)